addons/mail/static/src/js/chatter.js

```diff
@@ -158,6 +159,17 @@ var Chatter = Widget.extend({
         });
         this._updateAttachmentCounter();
     },
+    /**
+     * resets _suggestedPartnersProm and call _openComposer
+     * if composer is already open, to update suggested recipients
+     */
+    async updateSuggestedPartners() {
+        this._resetSuggestedPartners();
```

Comment:
why not turn `_resetSuggestedPartners` into this `updateSuggestedPartners`, so that any flow that reset suggested recipient goes through this logic?

Reply:
@alexkuhn the reason why I did not turn ```_resetSuggestedPartners``` into ```updateSuggestedPartners``` is, sometimes we need only reset of ```_suggestedPartnersProm``` for e.g. when message is posed we just want to reset ```_suggestedPartnersProm```  when messaged is posted reset of suggested partner is called before composer is closed and due to which it calls ```get_suggested_recipients``` which is useless call until composer is opened again.